pennylane/ops/identity.py

```diff
@@ -60,6 +60,10 @@ def _heisenberg_rep(p):
     def diagonalizing_gates(self):
         return []
 
+    @staticmethod
+    def compute_decomposition(data, wires, hyperparameters):
```

Comment:
woah that is a lot of pylint warnings

Reply:
That was my first pass signature.  Updating it to unpacked version now.